src/mode/buffered_graphics.rs

```diff
@@ -0,0 +1,227 @@
+//! Buffered graphics mode.
+
+use crate::{command::AddrMode, rotation::DisplayRotation, size::DisplaySize, Ssd1306};
+use display_interface::{DisplayError, WriteOnlyDataCommand};
+use generic_array::GenericArray;
+
+/// Buffered graphics mode.
+///
+/// This mode keeps a pixel buffer in system memory, up to 1024 bytes for 128x64px displays. This
+/// buffer is drawn to by [`set_pixel`](Ssd1306::set_pixel) commands or
+/// [`embedded-graphics`](https://docs.rs/embedded-graphics) commands. The display can then be
+/// updated using the [`flush`](Ssd1306::flush) method.
+#[derive(Clone, Debug)]
+pub struct BufferedGraphicsMode<SIZE>
+where
+    SIZE: DisplaySize,
+{
+    buffer: GenericArray<u8, SIZE::BufferSize>,
```

Comment:
Time for const generics, maybe?

Reply:
Yes definitely! I decided I want to do them in another PR to keep this one a little shorter.